client/containers/Pub/SpubHeader/SpubHeaderToolBar/spubHeaderToolbar.scss

```diff
@@ -0,0 +1,40 @@
+@import 'styles/variables.scss';
+
+.spubheader-toolbar {
+	display: flex;
+	justify-content: center;
+	align-items: baseline;
+	column-gap: 340px;
```

Comment:
Let's make our horizontal separation of the left and right elements more robust to different container sizes. Right now you have:
```css
justify-content: center;
column-gap: 340px;
```
This gets some space between the two sides, but it's always exactly `340px`, regardless of the size of the container. Where does that number come from? My hunch is that it emerges from your screen size, and it won't look right on other displays :) Instead what you want is:
```css
justify-content: space-between
```
to put as much space as possible between the two children.